src/lib/construct-line.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+function constructLines(line) {
+  const lines = [];
+  line.clusters_x.map((point, i) => {
```

Comment:
Do we need to account for `line` being null? (Or `line.clusters_x`, `line.cluster_y`) I don't see any safety checks for the data format prior to this function call, and we'd need some failsafes to ensure the basic transcription functions still work if, e.g. Caesar inadvertantly changes the data format.